alphadia/workflow/optimization.py

```diff
@@ -296,23 +302,26 @@ def _just_converged(self):
         if len(self.history_df) < 3:
             return False
 
-        if self.favour_narrower_parameter:  # This setting can be useful for optimizing parameters for which many parameter values have similar feature values.
+        if self.try_narrower_values:  # This setting can be useful for optimizing parameters for which many parameter values have similar feature values.
             min_steps_reached = (
                 self.num_prev_optimizations
                 >= self.workflow.config["calibration"]["min_steps"]
             )
 
             feature_history = self.history_df[self.feature_name]
             feature_substantially_decreased = (
-                feature_history.iloc[-1]
-                < self.maximal_decrease * feature_history.iloc[-2]
-                and feature_history.iloc[-1]
-                < self.maximal_decrease * feature_history.iloc[-3]
-            )
+                feature_history.iloc[-1] - feature_history.iloc[-2]
+            ) / np.abs(feature_history.iloc[-2]) < -self.maximal_decrease and (
```

Comment:
so `self.maximal_decrease` carries a minus sign? could we get rid of that (direction is already encoded in the name, from which I would also guess that it is always the same sign?)

Reply:
What do you mean by getting rid of it? I could force the use to use a negative sign in the config definition, but I thought it was easier to get the user to put in a positive value and then take the negative here. I am happy to change it if you think the former is more intuitive though.

Reply:
ah, I did not read the full expression 
`    (feature_history.iloc[-1] - feature_history.iloc[-2]) / np.abs(feature_history.iloc[-2]) < -self.maximal_decrease`
having `maximal_decrease` positive is fine ..

mb turn it around:
   ` ( feature_history.iloc[-2] - feature_history.iloc[-1]) / np.abs(feature_history.iloc[-2]) > self.maximal_decrease`

Reply:
I could do that. But I think it makes it harder to understand rather than easier (at least, I mentally reverse the operation in my head when trying to figure out what the code is doing).